### ISSUE TITLE
Use Github template when creating a PR

### ISSUE THREAD

**Issue by spyderrsh** [TeamCodeStream/codestream#479]
When creating a PR for GitHub, they have this system to automatically populate the PR using a file within the repo using [a template file](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/creating-a-pull-request-template-for-your-repository). I know that our team would be interested in using this functionality within codestream.

**Comment by planteater** [comment]
@spyderrsh - CodeStream does actually support template files, but it sounds like you're not seeing the template implemented when you create a PR? Can you let me know the name (including case) and location of your template file? Thanks!

**Comment by spyderrsh** [comment]
@planteater Wow how did I miss that! I clicked create PR on the wrong instance of Android Studio and kind of just assumed it wasn't it an option. My team will be ecstatic to know that!

BTW I opened an issue about GitHub PRs about a year ago. My team at that time decided to move away from CodeStream. I just switched to a new team and we started looking into CodeStream again and all the work you did with PRs is MAGIC!!! Thank you soo much! It looks great!
